Features/Fine/FineUserInterface/DemoApp/Sources/FineUserInterfaceAppDelegate.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FineAppDelegate.swift
+//
+//  Fine
+//
+//  Created by hyerin on .
+//
+
+import UIKit
+
+import FineUserInterface
+import FineUserInterfaceImpl
+
+import RIBs
+
+@main
+final class FineAppDelegate: UIResponder, UIApplicationDelegate {
+	
+	private final class MockFineDependency: FineListDependency { }
+	
+	private final class MockFineListener: FineListListener { }
+	
+    var window: UIWindow?
+	
+	private var router: ViewableRouting?
+	
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let window = UIWindow(frame: UIScreen.main.bounds)
+        
+		router = FineListBuilder(dependency: MockFineDependency())
+			.build(withListener: MockFineListener())
+		router?.load()
+		router?.interactable.activate()
```

Comment:
interactor가 activate되고 load되는 것이 일반적 루트입니다. (이 코드를 추가하는 이유는 저희가 attachChild를 임의로 해서 동작하게함이라서 attachChild내부에서 하는 순서랑 동일하게 맞추면 좋아요 :-) )
```swift
// We cannot declare the attach/detach child methods to take in concrete `Router` instances,
    // since during unit testing, we need to use mocked child routers.

    /// Attaches the given router as a child.
    ///
    /// - parameter child: The child `Router` to attach.
    public final func attachChild(_ child: Routing) {
        assert(!(children.contains { $0 === child }), "Attempt to attach child: \(child), which is already attached to \(self).")

        children.append(child)

        // Activate child first before loading. Router usually attaches immutable children in didLoad.
        // We need to make sure the RIB is activated before letting it attach immutable children.
        child.interactable.activate()
        child.load()
    }
```